third_party/color_emoji/emoji_builder.py

```diff
@@ -491,14 +497,20 @@ def is_vs(cp):
 			codes = img_file[len (img_prefix):-4]
 			if "_" in codes:
 				pieces = codes.split ("_")
-                                cps = [int(code, 16) for code in pieces]
-				uchars = "".join ([unichr(cp) for cp in cps if not is_vs(cp)])
+				cps = [int(code, 16) for code in pieces]
+				if sys.version_info >= (3, 0, 0):
+					uchars = "".join ([chr(cp) for cp in cps if not is_vs(cp)])
+				else:
+					uchars = "".join ([unichr(cp) for cp in cps if not is_vs(cp)])
```

Comment:
just make an global alias at the top of this module like this below, instead of checking sys.version all the time

```python
try:
    unichr  # py2
except NameError: 
    unichr = chr  # py3
```